### ISSUE TITLE
wanted database.sql file

### ISSUE THREAD

**Issue by jasho007** [Devsorb/Restaurant-Webapp#2]
please give the file we want to make project

**Comment by asifkhn143** [comment]
Please add the database data from mysql. import it and save it here.

**Comment by nitishpatel** [comment]
hey you literally uploaded every thing but not the database.sql file

**Comment by Prakruthi19** [comment]
Please upload database.sql file. This project is everything I needed. If you upload sql file it will be really helpful

**Comment by botstylee** [comment]
How come the code for the order is not complete

**Comment by GamxXamg** [comment]
Anyway to get the database SQL ? Thanks for sharing the code but with the sql that would be better :D

**Comment by densenocj** [comment]
lol, where's the *.sql file???

**Comment by rachid123RA** [comment]
sql file ??????

**Comment by densenocj** [comment]
Yes, there's no DB file to start the application i cannot test it, please
help

Thanks

Carlos Jonathan Castro Méndez
***@***.*** / +525543459677

________________________________________________________________________


Bill Gates' dreams are constantly interrupted by blue screens?
